Mobile/ShopLens/ImageRecognitionMobile/RecognitionResult.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ImageRecognitionMobile
+{
+    public class RecognitionResult
+    {
+        public Dictionary<string, float> Predictions { get; set; } = new Dictionary<string, float>();
+        public string WeightSpecifier { get; set; } = "";
+        public string RawOcrResult { get; set; } = "";
```

Comment:
I also started using "" for empty strings today.

Reply:
Some say `string.Empty()` is more readable